docs/tools.rst

```diff
@@ -225,6 +225,37 @@ Example
       -fpoints [[283,204,105]] \
       -output tmp_out.nii.gz
 
+Inference
+------------
+
+Provides implementation for ``nvidia::aiaa::Client::inference()`` API.
+For more details refer `aiaa-inference.cpp <https://github.com/NVIDIA/ai-assisted-annotation-client/blob/master/cpp-client/tools/aiaa/aiaa-inference.cpp>`_
```

Comment:
file is not there, this is just placeholder so we check if this link is working later